test/udev-test.pl

```diff
@@ -1558,6 +1558,11 @@ sub udev_setup {
 
         system("rm", "-rf", "$udev_run");
 
+        if (!mkdir($udev_run)) {
+                warn "unable to create directory $udev_dev";
```

Comment:
I think `$udev_run` should be here instead of `$udev_dev`.

Reply:
Thanks! I've corrected that.